tests/std/tests/LWG3515_stacktrace_ostream_operator_should_be_less_templatized/test.compile.pass.cpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <concepts>
+#include <memory>
+#include <memory_resource>
+#include <ostream>
+#include <sstream>
+#include <stacktrace>
+#include <string>
+
+using namespace std;
+
+template <class Ostream, class Alloc = allocator<stacktrace_entry>>
+concept CanPrintStacktrace = requires(Ostream& os, const stacktrace_entry& f, const basic_stacktrace<Alloc>& st) {
+                                 os << f;

```

Comment:
I suggest requiring the correct type and value category for these expressions (a la `{ os << f } -> same_as<Ostream&>;`) as well. Doing so would justify including `<concepts>`, which the test otherwise doesn't need ;)

Reply:
> a la `{ os << f } -> same_as<Ostream&>;`

That was my initial intention, done!